routes/index.js

```diff
@@ -57,6 +57,11 @@ module.exports = function(passport){
     res.send("This is your homepage!");
   });
 
+  /*Nunjucks rendering for upload page*/
+  router.get('/upload',function(req,res){
```

Comment:
`router.get('/upload'...` already exists in the file. Please either delete the other one or this one and modify the existing one.